common/test/unit/common/test_file_utils.py

```diff
@@ -1,7 +1,8 @@
 import unittest
 import os
+import mock
```

Comment:
This needs a blank line above for pep8, since ``mock`` in python 2 is not part of the standard library.